docs/aggregation.md

```diff
@@ -1,12 +1,34 @@
-# qgrid 
+# qgrid
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows='data'>
+	<q-grid:columns>
+		<q-grid:column key="Name" title="Name" aggregation="count"></q-grid:column>
+		<q-grid:column key="Age" title="Age" aggregation="avg"></q-grid:column>
+		<!-- max || min || sum || count || avg -->
+	</q-grid:columns>
+	<!-- naming: aggregation, aggregation-footer -->
+	<q-grid:aggregation></q-grid:aggregation>
```

Comment:
I'm not sure about this. If we're talking about aggregation of the whole dataset (e.g. count or avg by all collection, we don't need any aggregation at all since we have `footer` component.
But if we need some `group by`-like functionality, we definitely need templates for aggregation